packages/plugins/experimental/plugin-calls/src/utils/get-device-list.ts

```diff
@@ -0,0 +1,6 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+// TODO(mykola): Handle devicechange event.
+export const getDeviceList = () => navigator.mediaDevices.enumerateDevices();
```

Comment:
better to have fewer util files and co-locate logic. consolidate some of the get-X files?